### ISSUE TITLE
Testing out the artifact alops step but get: The following commands are already available on this system

### ISSUE THREAD

**Issue by MortenRa** [HodorNV/ALOps#181]
should the parameter "s use -AllowClobber parameter " by at my end or at your end.

Build agent is running on same server as the steps without artifacts.

**the used yaml**

```yaml
trigger:
  branches:
    include:
    - master
  paths:
    include:
    - app/*
    - test/*

name: $(Build.BuildId)

pool: default

variables:
- group: Secrets
- name: 'AppVersion'
  value: '16.1.[yyyyWW].*'

steps:
- checkout: self
  clean: true 

- task: ALOpsDockerCreate@1
  displayName: 'ALOPS - Create Docker Image' # It will reuse the image if already exists, or build one if it doesn't.
  inputs:
    artifacttype: Sandbox

- task: ALOpsDockerStart@1
  displayName: 'ALOPS - Start Docker Container' # No need to provide any details - it will get the details from previous step

- task: ALOpsDockerWait@1
  displayName: 'ALOPS - Wait for Docker Container to start'
  inputs:
    search_string: 'Ready for connections!'

#########################################################################
- task: ALOpsLicenseImport@1
  displayName: 'ALOps License Import'
  inputs:
    usedocker: true
    license_path: $(LicenseFile)
    
- task: ALOpsAppCompiler@1
  displayName: 'ALOps Compile Extension: App'
  inputs:
    usedocker: true
    targetproject: 'app/app.json'
    nav_app_version: '$(AppVersion)'
    failed_on_warnings: false
    app_file_suffix: 'App'

- task: ALOpsAppSign@1
  displayName: 'ALOps App Sign'
  env:
    pfx_password: $(CodeSignPfxPassword)
  inputs:
    pfx_path: $(CodeSignPfxFile)
    nav_artifact_app_filter: '*app.app'
    timestamp_uri: 'http://timestamp.comodoca.com/authenticode'
    usedocker: True  

- task: ALOpsAppSignVerify@1
  displayName: 'ALOps App Sign Verify'
  inputs:
    usedocker: True
    nav_artifact_app_filter: '*app.app '

- task: ALOpsAppPublish@1
  displayName: 'ALOps Publish Extension'
  inputs:
    usedocker: true
    nav_artifact_app_filter: '*app.app'
    skip_verification: false   

- task: ALOpsAppRuntimePackage@1
  inputs:
    usedocker: true
    targetproject: 'app/app.json'
    publish_artifact: true
#########################################################################

- task: ALOpsDockerRemove@1
  displayName: 'ALOPS - Remove Docker Container'
  enabled: true
  condition: always()
```

**the output**
```
2020-08-25T11:58:15.5411910Z ##[section]Starting: ALOPS - Create Docker Image
2020-08-25T11:58:15.5551118Z ==============================================================================
2020-08-25T11:58:15.5551627Z Task         : ALOps Docker Create
2020-08-25T11:58:15.5552000Z Description  : Create Docker image from Business Central Artifacts
2020-08-25T11:58:15.5552342Z Version      : 1.433.1659
2020-08-25T11:58:15.5552596Z Author       : Hodor
2020-08-25T11:58:15.5552936Z Help         : Create Docker image based on NAV/BC Artifacts.
2020-08-25T11:58:15.5553347Z ==============================================================================
2020-08-25T11:58:17.4626660Z *** Validate configuration
2020-08-25T11:58:18.9224174Z *** ALOps License: ***
2020-08-25T11:58:19.2361271Z *** ALOps License: 
2020-08-25T11:58:19.2362481Z 
2020-08-25T11:58:19.2363112Z collectionUID : 1
2020-08-25T11:58:19.2363775Z projectUID    : 
2020-08-25T11:58:19.2364320Z licenseUID    : 
2020-08-25T11:58:19.2364813Z description   : 
2020-08-25T11:58:19.2365125Z 
2020-08-25T11:58:19.2365306Z 
2020-08-25T11:58:19.2365479Z 
2020-08-25T11:58:19.2398927Z 
2020-08-25T11:58:19.3666826Z *** Importing required PS-Functions
2020-08-25T11:58:25.0017142Z *** Install/Update BCContainerHelper [https://github.com/microsoft/navcontainerhelper]
2020-08-25T11:58:55.0993752Z ##[error]The following commands are already available on this system:'Add-FontsToBCContainer,Backup-BCContainerDatabases,Compile-AppInBCContainer,Copy-CompanyInBCContainer,Copy-FileFromBCContainer,Copy-FileToBCContainer,Create-AadAppsForBC,Create-AadUsersInBCContainer,Create-AlProjectFolderFromBcContainer,Enter-BCContainer,Export-BCContainerDatabasesAsBacpac,Extract-FilesFromBCContainerImage,Extract-FilesFromStoppedBCContainer,Get-BCContainerApiCompanyId,Get-BCContainerApp,Get-BCContainerAppInfo,Get-BCContainerAppRuntimePackage,Get-BCContainerBCUser,Get-BCContainerCountry,Get-BCContainerDebugInfo,Get-BCContainerEula,Get-BCContainerEventLog,Get-BCContainerGenericTag,Get-BCContainerId,Get-BCContainerImageLabels,Get-BCContainerImageName,Get-BCContainerImageTags,Get-BCContainerIpAddress,Get-BCContainerLegal,Get-BCContainerName,Get-BCContainerNavVersion,Get-BCContainerOsVersion,Get-BCContainerPath,Get-BCContainerPlatformVersion,Get-BCContainers,Get-BCContainerServerConfiguration,Get-BCContainerSession,Get-BCContainerSharedFolders,Get-BCContainerTenants,Get-BestBCContainerImageName,Get-CompanyInBCContainer,Get-TestsFromBCContainer,Import-BCContainerLicense,Import-ConfigPackageInBCContainer,Import-PfxCertificateToBCContainer,Import-TestToolkitToBCContainer,Install-BCContainerApp,Install-NAVSipCryptoProviderFromBCContainer,Invoke-BCContainerApi,Invoke-ScriptInBCContainer,New-BCContainer,New-BCContainerBCUser,New-BCContainerTenant,New-BCContainerWindowsUser,New-CompanyInBCContainer,Open-BCContainer,Publish-BCContainerApp,Publish-NewApplicationToBcContainer,Remove-BCContainer,Remove-BCContainerSession,Remove-BCContainerTenant,Remove-CompanyInBCContainer,Remove-ConfigPackageInBCContainer,Repair-BCContainerApp,Replace-BCServerContainer,Restart-BCContainer,Restore-DatabasesInBCContainer,Run-TestsInBCContainer,Setup-BCContainerTestUsers,Setup-TraefikContainerForBCContainers,Sign-BCContainerApp,Start-BCContainer,Start-BCContainerAppDataUpgrade,Stop-BCContainer,Sync-BCContainerApp,Test-BCContainer,UnInstall-BCContainerApp,UnPublish-BCContainerApp,Wait-BCContainerReady,Add-FontsToNavContainer,Add-GitToAlProjectFolder,Backup-NavContainerDatabases,Check-NavContainerHelperPermissions,Clean-BcContainerDatabase,Compile-AppInNavContainer,Compile-ObjectsInNavContainer,Convert-AlcOutputToAzureDevOps,Convert-ModifiedObjectsToAl,Convert-Txt2Al,Copy-AlSourceFiles,Copy-CompanyInNavContainer,Copy-FileFromNavContainer,Copy-FileToNavContainer,Create-AadAppsForNav,Create-AadUsersInNavContainer,Create-AlProjectFolderFromNavContainer,Create-MyDeltaFolder,Create-MyOriginalFolder,Download-File,Enter-NavContainer,Export-ModifiedObjectsAsDeltas,Export-NavContainerDatabasesAsBacpac,Export-NavContainerObjects,Extract-AppFileToFolder,Extract-FilesFromNavContainerImage,Extract-FilesFromStoppedNavContainer,Flush-ContainerHelperCache,Generate-SymbolsInNavContainer,Get-BestNavContainerImageName,Get-CompanyInNavContainer,Get-LocaleFromCountry,Get-NavContainerApiCompanyId,Get-NavContainerApp,Get-NavContainerAppInfo,Get-NavContainerAppRuntimePackage,Get-NavContainerCountry,Get-NavContainerDebugInfo,Get-NavContainerEula,Get-NavContainerEventLog,Get-NavContainerGenericTag,Get-NavContainerId,Get-NavContainerImageLabels,Get-NavContainerImageName,Get-NavContainerImageTags,Get-NavContainerIpAddress,Get-NavContainerLegal,Get-NavContainerName,Get-NavContainerNavUser,Get-NavContainerNavVersion,Get-NavContainerOsVersion,Get-NavContainerPath,Get-NavContainerPlatformVersion,Get-NavContainers,Get-NavContainerServerConfiguration,Get-NavContainerSession,Get-NavContainerSharedFolders,Get-NavContainerTenants,Get-NavVersionFromVersionInfo,Get-TestsFromNavContainer,Import-ConfigPackageInNavContainer,Import-DeltasToNavContainer,Import-NavContainerLicense,Import-ObjectsToNavContainer,Import-PfxCertificateToNavContainer,Import-TestToolkitToNavContainer,Install-NavContainerApp,Install-NAVSipCryptoProviderFromNavContainer,Invoke-NavContainerApi,Invoke-NavContainerCodeunit,Invoke-ScriptInNavContainer,New-CompanyInNavContainer,New-DesktopShortcut,New-LetsEncryptCertificate,New-NavContainer,New-NavContainerNavUser,New-NavContainerTenant,New-NavContainerWindowsUser,Open-NavContainer,Publish-NavContainerApp,Publish-NewApplicationToNavContainer,Remove-CompanyInNavContainer,Remove-ConfigPackageInNavContainer,Remove-DesktopShortcut,Remove-NavContainer,Remove-NavContainerSession,Remove-NavContainerTenant,Renew-LetsEncryptCertificate,Repair-NavContainerApp,Replace-DependenciesInAppFile,Replace-NavServerContainer,Restart-NavContainer,Restore-DatabasesInNavContainer,Run-TestsInNavContainer,Setup-NavContainerTestUsers,Setup-TraefikContainerForNavContainers,Sign-NavContainerApp,Sort-AppFoldersByDependencies,Start-NavContainer,Start-NavContainerAppDataUpgrade,Stop-NavContainer,Sync-NavContainerApp,Test-NavContainer,UnInstall-NavContainerApp,UnPublish-NavContainerApp,Wait-NavContainerReady'. This module 'BcContainerHelper' may override the existing commands. If you still want to install this module 'BcContainerHelper', use -AllowClobber parameter.
2020-08-25T11:58:55.1459045Z ##[section]Finishing: ALOPS - Create Docker Image
```

**Comment by pri-kise** [comment]
You have to uninstall navcontainerhelper on the build agent at first

**Comment by waldo1001** [comment]
Exactly.

@MortenRa , could you please check if you have uninstalled NavContainerHelper?

**Comment by MortenRa** [comment]
Well I did not as my other pipelines is still running the "old  way" , this was to test with artifact so I guess I can not run both I have to switch to BCContainerHelper

**Comment by waldo1001** [comment]
You do - or you can simply install a (temporary) new build server?

**Comment by AdminHodor** [comment]
Dear @MortenRa,

Thanks for the feedback, we combined the "-AllowClober" with using Fully Qualified CmdLets for the next release (hotfix).
This will still allow for NavContainerHelper on the same system, but will ensure the correct CmdLets beeing used for ALOps.
We'll release the hotfix on this release asap and keep you posted on this.

Kind regards,

**Comment by AdminHodor** [comment]
Dear @MortenRa , @pri-kise ,

Should be fixed with out hotfix release v1.434.1672 available now.

Kind regards

**Comment by MortenRa** [comment]
Works fine thanks
